css/signup.css

```diff
@@ -1,107 +1,121 @@
 * {
-  margin:0;
-  padding:0;
+  margin: 0;
+  padding: 0;
   box-sizing: border-box;
 }
 body {
-  background-color:#f7f4f4;
-  font-family:sans-serif;
+  background-color: #f7f4f4;
+  font-family: sans-serif;
 }
-.blue {
-  color:#1F59BB;
-  text-transform: capitalize;
-  font-weight: 400;
+#signUpWrapper {
+  display: flex;
+  margin-top: 2.5%;
+  margin-left: 8.5%;
+}
+/**SignUpInfo**/
+#signUpInfo {
+  background: url("/blueBackground.png");
+  width: auto;
+  height: 600px;
+  border: transparent;
+  padding: 30px 50px;
+}
+.img#logo {
+  width: 90px;
+  height: 60px;
 }
-.info {
-  font-size: 13px;
-  font-weight: 600;
+#signUpInfo .welcome,
+#signUpInfo .welcomeSignUp {
   text-transform: capitalize;
+  color: #fff;
+  margin-top: 12px;
+  font-weight: 300;
 }
-.signUp {
-  padding:0rem 0.2rem;
+.welcomeSignUp {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".welcomeSignUp" to come before selector "#signUpInfo .welcomeSignUp" (no-descending-specificity)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5651419)